testdata/zk/Cheatcodes.t.sol

```diff
@@ -157,6 +164,14 @@ contract ZkCheatcodesTest is DSTest {
         new Emitter();
     }
 
+    function testExpectEmitIgnoresStaticCalls() public {
```

Comment:
I don't understand this test. Where is the static call happening and how is expect emit ignoring it?

Reply:
`emitter.FUNCTION_MESSAGE()`, due to it being a public field, would be the static call.
In general, view functions are static calls - perhaps it should be made clearer if `FUNCTION_MESSAGE` wasn't a constant, so the compiler couldn't optimize away the read